torchaudio/prototype/ctc_decoder/ctc_decoder.py

```diff
@@ -257,3 +264,57 @@ def lexicon_decoder(
         sil_token=sil_token,
         unk_word=unk_word,
     )
+
+
+def download_pretrained_files(
+    path: Union[str, Path],
+    dataset: str,
+    lm: Optional[str] = None,
+) -> PretrainedFiles:
```

Comment:
I am wondering if it is necessary to make `path` explicit.
Since the implementation is using hub download feature, how about abstracting away the path where the model is downloaded, like `pipelines` module does?